src/Mod/Path/PathScripts/post/grbl_post.py

```diff
@@ -340,20 +340,20 @@ def export(objectslist, filename, argstring):
 
 
 def linenumber():
+  if not OUTPUT_LINE_NUMBERS:
+    return ""
```

Comment:
The entire indentation of this file is of 2 spaces per level of indentation. To comply with PEP8 standards, it should be of 4 spaces. Hopefully it can be changed in the future.

Reply:
how can that be done automatically?

Reply:
You can probably run a code formatter, but I don't trust them, so I would do it manually, and at the same time I'd clean up many things.